example/src/main/java/com/ryanharter/auto/value/gson/example/PersonTypeAdapterFactory.java

```diff
@@ -0,0 +1,18 @@
+package com.ryanharter.auto.value.gson.example;
+
+import com.google.gson.Gson;
+import com.google.gson.TypeAdapter;
+import com.google.gson.TypeAdapterFactory;
+import com.google.gson.reflect.TypeToken;
+
+public class PersonTypeAdapterFactory implements TypeAdapterFactory {
```

Comment:
Sorry I just return to my laptop. How could we register a type adapter without factory? The generated `GsonTypeAdapter` need a gson parameter for its constructor, but before we build gson, we have no gson instance.